### ISSUE TITLE
Response to Launch Requests after Metro Closing with Metro Closed message

### ISSUE THREAD

**Issue by pmyers88** [pmyers88/dc-metro-echo#46]
If you do a Launch Request after the Metro is closed it will still give you the normal message. Let's not waste the user's time.

`GetStation` requests do give an appropriate message when the Metro is closed.

**Comment by r33beers** [comment]
I don't think this is a blocker for releasing the app publicly. If you agree, please remove the release 1.0 milestone tag.
